src/main/java/com/onarandombox/MultiversePortals/PortalPlayerSession.java

```diff
@@ -257,24 +257,54 @@ public boolean showDebugInfo(MVPortal portal) {
     }
 
     private void showPortalPriceInfo(MVPortal portal) {
-        getPlayerFromName().sendMessage("More details for you: " + ChatColor.GREEN + portal.getDestination());
+        getPlayerFromName().sendMessage("More details: " + ChatColor.GREEN + portal.getDestination());
```

Comment:
Not really a point is showing this `More details` anymore since is just the destination, which is shown the line before in command output.

Reply:
Yeah, I thought it would give a reason why this was in there before - I will remove it.